src/clients/NEOClient.ts

```diff
@@ -74,24 +74,23 @@ export class NEOClient {
     const tokenQueryResults = await sdk.token.getAllTokens();
     const account = new Neon.wallet.Account(address);
     const tokens = tokenQueryResults.filter(
-      (token) =>
-        {
-          const isCorrectBlockchain = 
-          version === "V2" 
-            ? 
-            !!sdk.token.getBlockchainV2(token.denom) && (BLOCKCHAIN_V2_TO_V1_MAPPING[sdk.token.getBlockchainV2(token.denom)!] == this.blockchain) 
-            : 
+      (token) => {
+        const isCorrectBlockchain =
+          version === "V2"
+            ?
+            !!sdk.token.getBlockchainV2(token.denom) && (BLOCKCHAIN_V2_TO_V1_MAPPING[sdk.token.getBlockchainV2(token.denom)!] == this.blockchain)
+            :
             blockchainForChainId(token.chainId.toNumber(), sdk.network) == this.blockchain
-          return (isCorrectBlockchain || token.denom === "swth") && token.tokenAddress.length == 40 && token.bridgeAddress.length == 40
-        }
+        return (isCorrectBlockchain || token.denom === "swth") && token.tokenAddress.length == 40 && token.bridgeAddress.length == 40
+      }
     );
 
     const client: Neon.rpc.RPCClient = new Neon.rpc.RPCClient(url, "2.5.2"); // TODO: should we change the RPC version??
 
     // NOTE: fetching of tokens is chunked in sets of 15 as we may hit
     // the gas limit on the RPC node and error out otherwise
-    const promises: Promise<{}>[] = chunk(tokens, 75).map(async (partition: ReadonlyArray<Models.Token>) => { // tslint:disable-line
-      let acc: SimpleMap<string> = {};
+    const promises: Promise<any>[] = chunk(tokens, 75).map(async (partition: ReadonlyArray<Models.Token>) => { // tslint:disable-line
```

Comment:
Try not to use any. Think can just use `Promise<SimpleMap<string>>` since each promise returns `SimpleMap<string>`